### PR TITLE
Add put.io

### PR DESCRIPTION
**Pull request by ozhanefemeral**

### CODE REVIEW

File: img/p/put.io.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....032V158H164.273V161.75H156.753V165.5H149.234V169.25H134.195V173H96.5973V176.75H92.8376V195.5H89.0779V199.25H85.3181V203H66.5195V199.25H62.7597V195.5H59L59 94.2498ZM100.356 94.2501V86.7501H104.116V83.0001H111.635V79.2501H119.155V75.5001H130.434V79.2501H141.713V83.0001H145.473V86.7501H149.233V94.2501H152.993V116.75H149.233V120.5H145.473V124.25H141.713V128H137.954V131.75H111.635V128H107.876V124.25H104.116V120.5H100.356V113H96.5964V94.2501H100.356Z" fill="white"/><path d="M115.396 90.5H107.877V98H104.117V109.25H107.877V116.75H111.637V109.25H115.396V101.75H119.156V98H126.675V94.25H134.195V90.5H137.955V86.75H126.675V83H122.916V86.75H115.396V90.5Z" fill="#FDCE45"/><path d="M201.871 94.25H198.111V128H194.352V139.25H190.592V146.75H186.832V150.5H183.072V154.25H179.313V158H175.553V161.75H171.793V165.5H168.033V169.25H160.514V173H152.994V176.75H137.956V180.5H100.358L100.358 199.25H96.5984V203H92.8386V206.75H89.0789V210.5H77.7997V214.25H81.5594V218H100.358V214.25H104.118V210.5H107.878V191.75H111.637V188H149.235V184.25H164.274V180.5H171.793V176.75H179.313V173H183.072V169.25H186.832V165.5H190.592V161.75H194.352V158H198.111V150.5H201.871V139.25H205.631V101.75H201.871V94.25Z" fill="#FDCE45"/></svg>
```
Comment (hkamran80) [inline img/p/put.io.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 942 228"><defs><clipPath id="a"><path d="M0 567h822V0H0Z"/></clipPath></defs><g clip-path="url(#a)" transform="matrix(1.333 0 0 -1.333 -93 500)"><path fill-rule="evenodd" d="M594 375c-13 0-23-10-23-23 0-12 10-22 23-22s23 10 23 22c0 13-10 23-23 23m-82-25v7c0 9-7 16-17 16h-99c-9 0-16-7-16-16v-7c0-9 7-16 16-16h20c10 0 10-5 10-8v-74c0-9 7-16 16-16h7c9 0 16 7 16 16v74c0 3 0 8 11 8h19c10 0 17 7 17 16m9-69c-13 0-23-10-23-23s10-23 23-23c12 0 22 10 22 23s-10 23-22 23m76 30h-6c-10 0-17-7-17-16v-43c0-9 7-16 16-16h7c9 0 16 7 16 16v43c0 9-7 16-16 16m-252 63h-7c-4 0-8-2-11-5s-5-7-5-12v-52c0-16-12-30-26-30s-26 14-26 30v52c0 9-7 17-16 17h-6c-10 0-18-8-18-17v-53c0-39 28-69 65-69 38 0 67 30 67 69v53c0 9-8 17-17 17m-208-96a28 28 0 1 0 0 56 28 28 0 1 0 0-56m69 27c0 37-31 68-69 68s-68-30-68-69v-85c0-8 7-15 15-15h6c8 0 15 7 15 15v10c0 4 4 7 8 7h26c37 1 67 31 67 69m502-28a28 28 0 1 0 0 56c16 0 29-12 29-28 0-15-13-28-29-28m0 96c-38 0-68-30-68-69 0-38 29-68 67-68s69 31 69 69c0 37-31 68-68 68"/></g></svg>
```

File: entries/p/put.io.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "put.io": {
+    "domain": "put.io",
+    "tfa": [
+      "totp"
+    ],
+    "categories": [
+      "backup",
+      "entertainment"
```
Comment (hkamran80) [inline entries/p/put.io.json]:
```suggestion
      "backup"
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Comment by altaywtf** [comment]
@hkamran80 that is not the logo we want to use. actually, **that is not a logo at all**. it's just a text that says put.io 😅 

can you please revert to the one we've sent in the original pull request?

thanks! 🙏

**Pull request merged**